### ISSUE TITLE
Setting: Add to playlist instead of clear playlist and play selecetd album/song

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#75]
```
Kinda like a "partymode setting"; a setting where you would choose the default behavior
when pressing an album or song. 
3 settings possible: 1) Play now, 2) add to playlist, 3) play next. 

Reason: Many people to a party that always clears the current playlist can be very
annoying. I know that it is possible by pressing long on a number/song, but that can
be hard when people have gotten their first three tequilas... 

Thanks a lot in advance!
```

Original issue reported on code.google.com by `praestekjaer` on 2013-09-20 17:46:06

**Comment by kaaholst** [comment]
```
Hi and thanks for the suggestion.

I am happy to tell you this is already in the pipeline, and default actions for songs
and albums can be selected (via settings) in the next version of Squeezer, which will
hopefully be available soon.

```

Original issue reported on code.google.com by `kaaholst` on 2013-09-21 20:15:25

**Comment by kaaholst** [comment]
```
Thanks for the report.  We released 1.0.0 with this feature last week.  Go in to "Settings",
and choose the "Song select action".  Hope you like it.
```

Original issue reported on code.google.com by `nik@ngo.org.uk` on 2013-12-22 20:48:42
